admin-ui/app/utils/ApiResources.js

```diff
@@ -9,3 +9,4 @@ export const LDAP = 'ldap'
 export const SQL = 'sql'
 export const COUCHBASE = 'couchbase'
 export const CACHE = 'cache'
+export const USERS = 'user_management'
```

Comment:
why `user_management` varible here

Reply:
removed this one as this one is not used anywhere